src/geometry/GeometryEvaluator.cc

```diff
@@ -1034,10 +1034,19 @@ static Geometry *extrudePolygon(const LinearExtrudeNode& node, const Polygon2d&
       for (const auto& v : o.vertices)
         max_r1_sqr = fmax(max_r1_sqr, v.squaredNorm());
```

Comment:
I see you didn't edit this line, but maybe you could {} wrap these two for loops also while you're at it, since it's right there, or any others you see nearby.  (It's a good standard practice for the code to have those, in case someone goes to insert an indented line and misses that they are absent.)